libs/protocol-client/src/lib/utils/ipfs.ts

```diff
@@ -0,0 +1,71 @@
+import { Buffer } from 'buffer';
+import { create, IPFSHTTPClient, CID } from 'ipfs-http-client';
+import { ContributionV1SchemaCid } from './constants';
+
+export type MintedContributionSchemaV1 = {
+  schema: `ipfs://Qmc3fKseQ1JyiJiPsmazx23A6twy6YCkTfqDfHUKZDL5xq`;
+  name: string;
+  details: string;
+  proof: string;
+  activityName: string;
+  image?: string; // keep blank for now
```

Comment:
Thinking that this could potentially be a separate field for an image to allow for uploads of images since our proof field is more of a catchall?

Reply:
Image is more of like an NFT image to represent the contribution.